### ISSUE TITLE
fastqgz_to_counts.py broken after last merge

### ISSUE THREAD

**Issue by mschroed** [mhorlbeck/ScreenProcessing#12]
Spaces and new lines are breaking fastqgz_to_counts.py at lines 119-127 since the last merge for python3.

```bash
python --version
Python 3.6.8 :: Anaconda, Inc.

python fastqgz_to_counts.py -p 6 --trim_start 1 --trim_end 35 library_reference/CRISPRi_v1_human.trim_1_35.fa Demo/Step1/output Demo/Step1/Sequencing_files/Demo*
  File "fastqgz_to_counts.py", line 120
    if curSeqId != '' and curSeq != '':
                                      ^
TabError: inconsistent use of tabs and spaces in indentation
```

The commit in question is:
https://github.com/mhorlbeck/ScreenProcessing/pull/10/commits/57e875bb53f69aca496b4c1c81d4e379eaafb193#diff-1014e879b4e674cedbcbfae7dea982b0461bf9ad0434e7efa3e7c3fbd1b37005R119

Best,
M

**Comment by LonnekeNouwen** [comment]
Hi,
I have the same issue. Is there already a solution?
Best,
Lonneke

**Comment by mhorlbeck** [comment]
fixed with #13
